src/index.ts

```diff
@@ -45,16 +48,32 @@ pieces.forEach((piece) =>
 // All the color available in the game based on the pieces id available
 const COLORS = pieces.map((piece) => piece.id.split('-')[0]);
 
+let duplicate = true;
+let nbTurns = 7;
+let nbColors = 8;
+let nbPossibilities = 6;
```

Comment:
Done, typed too don't know if useful

Reply:
It wasn't necessary because typescript can type variable when they are declared, for example : 

`let a = 2`

typescript will automaticcaly type a as number !

Reply:
okay, perfect